pkg/daemons/agent/agent.go

```diff
@@ -197,7 +220,7 @@ func checkCgroups() (kubeletRoot, runtimeRoot string, hasCFS, hasPIDs bool) {
 				if _, err := os.Stat(p); err == nil {
 					hasCFS = true
 				}
-			} else if system == "name=systemd" {
+			} else if system == "name=systemd" || v2 {
```

Comment:
When on a non-systemd system but with unified cgroups, is it OK that we aren't setting `kubeletRoot` if we don't have a `.scope`?

Reply:
I'm not familiar with kubeletRoot stuff honestly l, but this PR corresponds to the v1 behavior.